lib/driver.js

```diff
@@ -393,15 +393,11 @@ class EspressoDriver extends BaseDriver {
     // start an avd, set the language/locale, pick an emulator, etc...
     // TODO with multiple devices we'll need to parameterize this
     await helpers.initDevice(this.adb, this.opts);
+
+    // default state is window animation disabled
+    this.caps.disableWindowAnimation = this.caps.disableWindowAnimation !== false;
```

Comment:
I don't see a need to override the cap value. It is enough to call `await this.setWindowAnimationState(this.caps.disableWindowAnimation !== false);`